pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/tissue/TissueDao.java

```diff
@@ -75,8 +78,30 @@ public Map<String, List<Tissue>> getTissuesByStudy(String study) {
             return execResult;
         });
         if (results.resultException != null) {
-            throw new RuntimeException("Error getting tissues for " + study, results.resultException);
+            throw new DsmInternalError("Error getting tissues for " + study, results.resultException);
         }
         return tissues;
     }
+
+    public static List<String> getTissuesByOncHistoryDetailId(String oncHistoryDetailId) {
+        List<String> tissueIds = new ArrayList<>();
+        SimpleResult results = inTransaction((conn) -> {
+            SimpleResult execResult = new SimpleResult();
+            try (PreparedStatement stmt = conn.prepareStatement(SQL_SELECT_TISSUES_BY_ONC_HISTORY_ID)) {
+                stmt.setString(1, oncHistoryDetailId);
+                try (ResultSet rs = stmt.executeQuery()) {
+                    while (rs.next()) {
+                        tissueIds.add(rs.getString(DBConstants.TISSUE_ID));
+                    }
+                }
+            } catch (SQLException ex) {
+                execResult.resultException = ex;
+            }
+            return execResult;
+        });
+        if (results.resultException != null) {
+            throw new DsmInternalError("Error getting tissue id by onc history detail id " + oncHistoryDetailId, results.resultException);
+        }
+        return tissueIds;
+    }
```

Comment:
This method is made to find the tissues that belong to a specific onc history. The idea behind this is if an onc history gets deleted, the corresponding tissues for it should also get deleted